cmd/build.go

```diff
@@ -193,7 +195,26 @@ func (b *buildCmd) createBuildTask() (*graph.Task, error) {
 	// TODO: create secrets
 	secrets := []*graph.Secret{}
 
-	return graph.NewTask(steps, secrets, b.opts.Registry, b.registryUser, b.registryPw, taskTotalTimeoutInSec, true)
+	var credentials []*graph.Credential
```

Comment:
nit: you can pre-allocate the array size rather than calling `append` multiple times